zk-index.el

```diff
@@ -526,11 +530,20 @@ Optionally refresh with FILES, using FORMAT-FN and SORT-FN."
   (other-window -1))
 
 (defun zk-index-view-note ()
-  "View note in view mode."
+  "View note while browsing zk-index in view mode."
   (interactive)
+  ;; Kill old viewing buffer
+  (if (get-buffer zk-index-view-buf)
+      (kill-buffer zk-index-view-buf))
+
+  (delete-other-windows)
```

Comment:
This function shouldn't delete windows or kill buffers, because a user might have `zk-index-auto-scroll` set to nil, in which case, it should just open the note in view-mode and leave the point in the zk-index or zk-desktop buffer.